TipCalculator/MainViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ViewController.swift
+//  TipCalculator
+//
+//  Created by vmalikov on 1/30/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+extension Float {
+    func format(f: String) -> String {
+        return NSString(format: "$%\(f)f", self) as String
+    }
+}
+
+class MainViewController: UIViewController, UITextFieldDelegate {
```

Comment:
Also, I use `// MARK:` to organize methods within the class.
// MARK: – Text Methods (to add a line)
// MARK: Text Methods (no line)
More details [here](http://www.learnswiftonline.com/reference-guides/organizing-your-swift-code/)